[2023-10-20 23:22:56,617] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-10-20 23:22:56,637] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-06-13T00:00:00+00:00 [queued]>
[2023-10-20 23:22:56,638] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:22:56,638] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-10-20 23:22:56,639] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-10-20 23:22:56,663] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2023-06-13 00:00:00+00:00
[2023-10-20 23:22:56,669] {standard_task_runner.py:52} INFO - Started process 3155 to run task
[2023-10-20 23:22:56,674] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2023-06-13T00:00:00+00:00', '--job-id', '957', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmp_edti7mf', '--error-file', '/tmp/tmpjs8jirop']
[2023-10-20 23:22:56,676] {standard_task_runner.py:80} INFO - Job 957: Subtask segunda_tarea
[2023-10-20 23:22:56,795] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2023-06-13T00:00:00+00:00 [running]> on host dfdda228c9fc
[2023-10-20 23:22:56,912] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2023-06-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-06-13T00:00:00+00:00
[2023-10-20 23:22:56,914] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-10-20 23:22:56,914] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-10-20 23:22:56,923] {subprocess.py:85} INFO - Output:
[2023-10-20 23:22:56,924] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-10-20 23:22:56,925] {subprocess.py:96} INFO - Command exited with return code 0
[2023-10-20 23:22:56,969] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20230613T000000, start_date=20231020T232256, end_date=20231020T232256
[2023-10-20 23:22:57,008] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-20 23:22:57,080] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
